.doc_gen/metadata/sts_metadata.yaml

```diff
@@ -66,9 +66,12 @@ sts_GetSessionToken:
 sts_Scenario_AssumeRoleMfa:
   title: Assume an &IAM; role that requires an MFA token with &STS; using an &AWS; SDK
   title_abbrev: Assume an &IAM; role that requires an MFA token
-  synopsis: assume an &IAM; role that requires a multi-factor authentication
-    (MFA) token with &STS; and use temporary security credentials to access an &AWS;
-    resource.
+  synopsis_list:
+    - Create an &IAM; role that grants permission to list &S3; buckets.
```

Comment:
&S3; --> S3

Reply:
fixed. also fixed other similar occurrences in this same file.